common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -610,6 +610,10 @@ public static int hashCodeAscii(CharSequence bytes) {
         return new MpscLinkedQueue<T>();
     }
 
+    public static <T> Queue<T> newSpscQueue() {
+        return new SpscLinkedQueue<T>();
```

Comment:
Check if there Unsafe is present and if not just return a `LinkedConcurrentQueue` (just like we do in `newMpscQueue()`.